### ISSUE TITLE
What needs to be done.

### ISSUE THREAD

**Issue by Magicode1** [DragonetMC/Dragonet#12]
What needs to be done? Are we just going to change the Glowstone software, to support PE clients? If so, what would you like me to do? 

I want to help, but at the same time I don't want to screw anything up :+1:

**Comment by DefinitlyEvil** [comment]
in my opinion we should wait for official PE server and start hacking on it to creating something like Bukkit.

**Comment by DaMatrix** [comment]
You'll end up getting a sofware in C++/#, making it nearly impossible to decompile and change.

**Comment by TheDiamondYT1** [comment]
@DaMatrix C# is easy to decompile. It will be in C++. A while ago shoghi tweeted that they were testing a server software and it was just the client code, *without the client code*. 

It will run on windows most likely and possibly mac or linux. No Android here folks! 

Its possible we could use DLL Injection to mod the server software

**Comment by DaMatrix** [comment]
Meh. C++ is nice and all, but I think it's gonna end up tearing the modding+server hosting community apart as it's mostly built on Java.

**Comment by DefinitlyEvil** [comment]
They are just too lazy to re-write a Java version. ;)

**Comment by TheDiamondYT1** [comment]
@DefinitlyEvil Not really. It makes sense. C++ server software for a C++ game. And if its C++, it can run as a native windows app

**Comment by VCraftMCPE** [comment]
@TheDiamondYT1 PE & PC togther  is impossible after 1.2 update because it requires Xbox live auth to join servers

**Comment by TheDiamondYT1** [comment]
Why did i just get notified aboiut this issue

**Comment by TheDiamondYT1** [comment]
@VCraftMCPE It's not impossible. 

Steve signs into xbox live
↓
Steve joins Dragonet server
↓
Steve enters his Minecraft PC account details in chat (or using a modal - MCPE 1.2 allows the server to display CUSTOM GUI!! @DefinitlyEvil)
↓
The required packets are sent to the PC server 
↓ 
Ta-da

**Comment by esmith715** [comment]
What?

**Comment by TheDiamondYT1** [comment]
What? 

Also — what?

**Comment by esmith715** [comment]
Its me TekkitCommando, also, why did you post that after so long?

**Comment by TheDiamondYT1** [comment]
@esmith715 VCraftMCPE commented and deleted his comment it shows in my feed

**Comment by esmith715** [comment]
Ah k
